tabbycat/motions/models.py

```diff
@@ -51,3 +47,25 @@ class Meta:
 
     def __str__(self):
         return "{0.motion.reference:s} ({0.preference:d}) by {0.debate_team!s}".format(self)
+
+
+class RoundMotion(models.Model):
+    """Represents the relation between rounds and motions"""
+
+    motion = models.ForeignKey(Motion, models.CASCADE,
+        verbose_name=_("motion"))
+    round = models.ForeignKey('tournaments.Round', models.CASCADE,
+        verbose_name=_("round"))
+
+    seq = models.IntegerField(default=1,
+        verbose_name=_("sequence number"),
+        help_text=_("The order in which motions are displayed"))
+
+    class Meta:
+        ordering = ('motion', 'seq')
```

Comment:
The `seq` is with respect to each round, not each motion, right? Wouldn't we want to order by `('round', 'seq')` rather than within motion?

Reply:
Thought about it myself.